.pylintrc

```diff
@@ -1,67 +1,33 @@
 [FORMAT]
 max-line-length = 120
+extension-pkg-whitelist=PyQt5
+
+[DESIGN]
+max-attributes=40  # TODO: refactor classes as needed to have default at 7
```

Comment:
Nice work arounds for now :)

Reply:
Yep, eventually we will have to refactor though :D